scripts/tests/check_partitions.rb

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env ruby
 # Test to see if expect partitions exist, fail if they don't
 EXPECTED_PARTITIONS = ['/var', '/var/log', '/var/log/audit', '/'].sort
-puts %x(findmnt --df)
+puts %x(df -h)
```

Comment:
Might want to strip out anything containing `tmpfs` here to make the output more readable.

Reply:
:ballot_box: